src/background/actions/action-payloads.ts

```diff
@@ -242,13 +242,22 @@ export interface UnifiedScanCompletedPayload extends BaseActionPayload {
 
 export interface RuleExpandCollapsePayload extends BaseActionPayload {
     ruleId: string;
+    testKey?: string;
```

Comment:
nit: I think it would be better to have 2 separate payloads here: a `RuleExpandCollapsePayload` that only has the ruleId that is used by the needs review and automated checks card selection store, and a separate `AssessmentSingleRuleExpandCollapsePayload` which **requires** both the ruleId and testKey. I think it's worthwhile to separate them out since I'm concerned that we would be confused about when the testKey is actually necessary if we have it as an optional parameter.